logic/peers.js

```diff
@@ -267,7 +327,8 @@ Peers.prototype.listRandomConnected = function(options) {
 		.map(key => self.peersManager.peers[key])
 		.filter(peer => peer.state === Peer.STATE.CONNECTED);
 	const shuffledPeerList = _.shuffle(peerList);
-	return options.limit ? shuffledPeerList.slice(0, options.limit)
+	return options.limit
+		? shuffledPeerList.slice(0, options.limit)
```

Comment:
options.limit is set to a list of peers?

Reply:
It's prittier's change, not mine. But yes, you can pass a limit to peers.list if that's your question.